src/Resources/Locales/de_DE.axaml

```diff
@@ -569,6 +595,7 @@
   <x:String x:Key="Text.Start" xml:space="preserve">START</x:String>
   <x:String x:Key="Text.Stash" xml:space="preserve">Stash</x:String>
   <x:String x:Key="Text.Stash.IncludeUntracked" xml:space="preserve">Inklusive nicht-verfolgter Dateien</x:String>
+  <x:String x:Key="Text.Stash.KeepIndex" xml:space="preserve">Behalte Dateien des Stages</x:String>
```

Comment:
We used "gestagte" before, e.g. `Text.Hotkeys.Repo.Commit`, `Text.Stash.OnlyStagedChanges`

```suggestion
  <x:String x:Key="Text.Stash.KeepIndex" xml:space="preserve">Behalte gestagte Dateien</x:String>
```